aspnet/mvc/views/razor.rst

```diff
@@ -1,8 +1,732 @@
+  :emphasize-lines: 5
+
+  @{
+      var joe = new Person("Joe", 33);
+   }
+
+  <p>Age@joe.Age</p>
+
+generates:
+
+.. code-block:: none
+
+  <p>Age@joe.Age</p>
+
+Razor is treating ``Age@joe.Age`` as an email alias. In cases like this, create an explicit expression with ``@()``:
+
+.. code-block:: none
+
+ <p>Age@(joe.Age)</p>
+
+Which renders
+
+.. code-block:: none
+
+  <p>Age33</p>
+
+.. _control-structures-razor-label:
+
+Control Structures
+------------------
+
+Control structures are an extension of code blocks. All aspects of code blocks (transitioning to markup, inline C#) also apply to the following structures.
+
+Conditionals ``@if``, ``else if``, ``else`` and ``@switch``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``@if`` family controls when code runs:
+
+.. code-block:: none
+
+  @if (value % 2 == 0)
+  {
+      <p>The value was even</p>
+  }
+
+``else`` and ``else if`` don't require the ``@`` symbol:
+
+.. code-block:: none
+
+ @if (value % 2 == 0)
+ {
+     <p>The value was even</p>
+ }
+ else if (value >= 1337)
+ {
+     <p>The value is large.</p>
+ }
+ else
+ {
+     <p>The value was not large and is odd.</p>
+ }
+
+``@switch``
+^^^^^^^^^^^^^
+
+.. code-block:: none
+
+ @switch (value)
+ {
+     case 1:
+         <p>The value is 1!</p>
+         break;
+     case 1337:
+         <p>Your number is 1337!</p>
+         break;
+     default:
+         <p>Your number was not 1 or 1337.</p>
+         break;
+ }
+
+Looping ``@for``, ``@foreach``, ``@while``, and ``@do while``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can render templated HTML with looping control statements. For example, to render a list of people:
+
+.. code-block:: none
+
+  @{
+      var people = new Person[]
+      {
+            new Person("John", 33),
+            new Person("Doe", 41),
+      };
+  }
+
+``@for``
+^^^^^^^^^
+
+.. code-block:: none
+
+  @for (var i = 0; i < people.Length; i++)
+  {
+      var person = people[i];
+      <p>Name: @person.Name</p>
+      <p>Age: @person.Age</p>
+  }
+
+``@foreach``
+^^^^^^^^^^^^
+
+.. code-block:: none
+
+  @foreach (var person in people)
+  {
+      <p>Name: @person.Name</p>
+      <p>Age: @person.Age</p>
+  }
+
+``@while``
+^^^^^^^^^^^^
+
+.. code-block:: none
+
+  @{ var i = 0; }
+  @while (i < people.Length)
+  {
+      var person = people[i];
+      <p>Name: @person.Name</p>
+      <p>Age: @person.Age</p>
+
+      i++;
+  }
+
+``@do while``
+^^^^^^^^^^^^^^^^
+
+.. code-block:: none
+
+  @{ var i = 0; }
+  @do
+  {
+      var person = people[i];
+      <p>Name: @person.Name</p>
+      <p>Age: @person.Age</p>
+
+      i++;
+  } while (i < people.Length);
+
+Compound ``@using``
+^^^^^^^^^^^^^^^^^^^^
+
+Compound using statements can be used to represent scoping. For instance, we can utilize :doc:`/mvc/views/html-helpers` to render a form tag with the ``@using`` statement:
+
+.. code-block:: none
+
+  @using (Html.BeginForm())
+  {
+      // Form content.
```

Comment:
I think you need to show some simple form content (just a text box and a submit button), so that it's clear that you can write more Razor content within the using statement block